lib/java.js

```diff
@@ -1,11 +1,13 @@
+var path = require('path');
 var java = require('java');
 
 java.asyncOptions = {
     syncSuffix: '',           // Synchronous methods the base name
     asyncSuffix: undefined,   // don't generate async (callback-style) wrappers
     promiseSuffix: undefined  // don't generate promise wrappers
-}
+};
 
-java.classpath.push('/Users/henridf/work/apache-spark-node/');
+// Parent folder
+java.classpath.push(path.normalize(__dirname+'/..'));
```

Comment:
As per my comment on the previous PR, using `path.join()` instead of `+` and `path.sep` instead of `/` is preferable. Though I take it this currently works on windows?